php/src/AstNode.php

```diff
@@ -31,11 +31,11 @@ public function add(RuleType $ruleType, object|array|string $entry): void
     /**
      * @template T of object
      *
-     * @param class-string<T> $expectedType
+     * @param class-string<T> $_expectedType
      *
      * @psalm-return list<T>
      */
-    public function getItems(string $expectedType, RuleType $ruleType): array
+    public function getItems(string $_expectedType, RuleType $ruleType): array
```

Comment:
I'm not familiar with php, what does the `_` do?

Reply:
It is a naming convention to mean "I'm not using this positional argument".

Reply:
Does php allow positional arguments to be named? I.e. would this break someone using this as a named argument?

Reply:
@mpkorstanje Oh I believed things about PHP which were true before 8.0.

https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments

> PHP 8.0.0 introduced named arguments as an extension of the existing positional parameters. Named arguments allow passing arguments to a function based on the parameter name, rather than the parameter position. This makes the meaning of the argument self-documenting, makes the arguments order-independent and allows skipping default values arbitrarily.

This makes my change wrong!

Reply:
Well, I followed the change tip in the Psalm note:

https://psalm.dev/docs/running_psalm/issues/UnusedParam/

> Can be suppressed by prefixing the parameter name with an underscore:

Reply:
Referring to the parameter, that works: 
411ea6d